src/swarm/neo/util/StateMachine.d

```diff
@@ -59,6 +59,8 @@ version ( UnitTest )
 
 *******************************************************************************/
 
+deprecated("Requests should be adapted to use one of the new request core "
+    "mixins in swarm.neo.client.mixin.")
```

Comment:
Can you put this on the same line, even if it goes over 80 chars ?

Reply:
Sure. Why?

Reply:
So it does not trigger a deprecation with the newest DMD (v2.071+)

Reply:
Gotcha.

Reply:
Updated.